charts/linkerd2/values.yaml

```diff
@@ -105,16 +105,21 @@ proxy:
   # - SMTP (25,587) server-first
   # - HTTPS (443) opaque TLS
   # - MYSQL (3306) server-first
+  # - Galera (4444) server-first
   # - PostgreSQL (5432) server-first
+  # - Redis (6379) server-first
+  # - ElasticSearch (9300) server-first
   # - Memcached (11211) clients do not issue any preamble, which breaks detection
-  opaquePorts: "25,443,587,3306,5432,11211"
+  opaquePorts: "25,443,587,3306,4444,5432,6379,9300,11211"
 
 # proxy-init configuration
 proxyInit:
-  # -- Default set of inbound ports to skip via itpables
-  ignoreInboundPorts: ""
-  # -- Default set of outbound ports to skip via itpables
-  ignoreOutboundPorts: ""
+  # -- Default set of inbound ports to skip via iptables
+  # - Galera (4567,4568) UDP
+  ignoreInboundPorts: "4567,4568"
+  # -- Default set of outbound ports to skip via iptables
+  # - Galera (4567,4568) UDP
+  ignoreOutboundPorts: "4567,4568"
```

Comment:
Note that UDP traffic is skipped by default by the proxy. According to Galera's docs, traffic on 4567 is TCP. I didn't see anything about 4568's protocol:
https://galeracluster.com/library/documentation/firewall-settings.html
https://galeracluster.com/library/documentation/glossary.html#term-cluster-replication
https://galeracluster.com/library/documentation/glossary.html#term-incremental-state-transfer
You can link to #5913 in these comments to keep a reference to the discussion about this case.

Reply:
Oh sorry, they just said TCP/UDP so I figured that's why we skipped them. I'll just remove the comment about why.